CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_fivetran_log v1.0.1
+
+## Bugfix:
+- Update staging models to current standard used in our other packages (the base, fields, final approach) ([PR #85](https://github.com/fivetran/dbt_fivetran_log/pull/85))
+- Rename CTEs in staging models to avoid potential circular references ([PR #85](https://github.com/fivetran/dbt_fivetran_log/pull/85))
```

Comment:
I think these two can be combined into a single entry. 

```suggestion
- Update staging models CTE names to current standard used in our other packages (the base, fields, final approach) to avoid potential circular references ([PR #85](https://github.com/fivetran/dbt_fivetran_log/pull/85))
```